src/browser/components/MainPage.jsx

```diff
@@ -369,6 +371,15 @@ export default class MainPage extends React.Component {
     this.handleSelect(key);
   };
 
+  handleInterTeamLink = (linkUrl) => {
+    const selectedTeam = Utils.getServer(linkUrl, this.props.teams);
+    if (!selectedTeam) {
+      return;
+    }
+    this.refs[`mattermostView${selectedTeam.index}`].handleDeepLink(linkUrl.href);
```

Comment:
Any chance the key could be undefined in `this.refs`?

Reply:
Shouldn't be, we do that check above to make sure that we find a team first. Then if `selectedTeam` is null that means we couldn't find one, so that's our null check.